sktime/forecasting/base/_fh.py

```diff
@@ -262,36 +386,48 @@ def to_numpy(self, **kwargs) -> np.ndarray:
         """
         return self.to_pandas().to_numpy(**kwargs)
 
+    def _coerce_cutoff_to_index_element(self, cutoff):
+        """Coerces cutoff to index element, and updates self.freq with cutoff."""
+        self.freq = cutoff
+        if isinstance(cutoff, pd.Index):
+            assert len(cutoff) > 0
+            cutoff = cutoff[-1]
```

Comment:
I guess we will see the new function here from https://github.com/alan-turing-institute/sktime/pull/2939, right?

Reply:
hm, we could use it, but then again this is only 3 lines?